delivery/media/dash/examples/mper/manifestmanipulation-1/README.md

```diff
@@ -0,0 +1,49 @@
+# Example
+The examples in this section detail how to use Dash module for manifest manipulation
+Manifest Personalization helps to dynamically create personalized versions of Dash manifests based on parameters like device type, user geography, request headers or query string parameters.This example demonstrates use of query string parameters.
+main.js provides examples using ReadableStream, WritableStream for different use cases as defined below:
+filterVariantsByBandwidth which filters out dash MPD representations by bandwidth value passed using query param br_in_range and br_in
+example curl :
+
+``` 
+curl -vk -H "Pragma: akamai-x-ew-debug-subs,akamai-x-ew-debug,akamai-x-ew-debug,akamai-x-ew-debug-rp" "https://hostname/sample1.mpd?br_in_range=400000-600000,800000-1000000"
```

Comment:
```suggestion
curl -vk "https://hostname/sample1.mpd?br_in_range=400000-600000,800000-1000000"
```

Deleted pragmas require enhanced debug.

I'm going to offer that as a suggest that for all of these but the code profiler specifically is especially bad to include here since it modifies the response body for responseProvider